src/lib/ui/thermostat/ThermostatCard.tsx

```diff
@@ -0,0 +1,149 @@
+import classNames from 'classnames'
+import { useState } from 'react'
+import {
+  isThermostatDevice,
+  type ThermostatDevice,
+  type ThermostatDeviceProperties,
+} from 'seamapi'
+
+import { FanIcon } from 'lib/icons/Fan.js'
+import { OffIcon } from 'lib/icons/Off.js'
+import { DeviceImage } from 'lib/ui/device/DeviceImage.js'
+import { ClimateSettingStatus } from 'lib/ui/thermostat/ClimateSettingStatus.js'
+
+interface ThermostatCardProps {
+  device: ThermostatDevice
+}
+
+export function ThermostatCard({ device }: ThermostatCardProps): JSX.Element {
+  return (
+    <div className='seam-thermostat-card'>
+      <Content device={device} />
+    </div>
+  )
+}
+
+function Content(props: { device: ThermostatDevice }): JSX.Element | null {
+  const { device } = props
+
+  const [temperatureUnit, setTemperatureUnit] = useState<
+    'fahrenheit' | 'celsius'
+  >('fahrenheit')
+
+  const toggleTemperatureScale = (): void => {
```

Comment:
I wonder if `scale` would have been a better API term than unit 🤔 

```suggestion
  const toggleTemperatureUnit = (): void => {
```